src/platforms/common/performance/troubleshooting-performance.mdx

```diff
@@ -9,7 +9,7 @@ supported:
 description: "Learn how to troubleshoot your performance monitoring setup."
 ---
 
-If you need help managing transactions, you can read more here. If you need additional help, please view our forums. Customers on a paid plan may also contact support.
+If you need help managing transactions, you can read more here. If you need additional help, you can ask on GitHub, and customers on a paid plan may also contact support.
```

Comment:
I feel like it might read better with the original sentence separation?

Reply:
Does this show up in both [Node.js](https://docs.sentry.io/platforms/node/guides/express/performance/troubleshooting-performance/) and [Python](https://docs.sentry.io/platforms/python/guides/tryton/performance/troubleshooting-performance/)?

Is that why we can't link to a specific GitHub repo here?

Reply:
Agreed, I kind of like the original sentence separation. This is on a page that's used by several platforms so yeah, we can't link to a specific repo here.

Reply:
Done in 1161c3cc87d36ff60e08b878f9bd068a00e7a300. 👍